docker/Dockerfile_puma

```diff
@@ -0,0 +1,41 @@
+FROM ruby:2.7
+ARG PORT_ARG=3009
+
+RUN apt-get update && apt-get install -y build-essential coreutils
+
+WORKDIR app
+
+COPY agent/* agent/
+COPY Gemfile .
+COPY Gemfile.lock .
+COPY vulneruby_engine.gemspec .
+COPY lib/vulneruby_engine/version.rb ./lib/vulneruby_engine/version.rb
+
+# Used in CI to install gem dropped in ./agent/ directory
+RUN gem install ./agent/contrast-agent.gem
+
+ENV PUMA=true
```

Comment:
Can you use this variable instead of having `PUMA_TEST`?

Reply:
`PUMA_TEST` follows the convention that Donnie set up before he left, so I went with that